app/packages/dataset/src/index.tsx

```diff
@@ -25,14 +25,39 @@ const DatasetWrapper = () => {
   );
 };
 
+const EXAMPLE_VIEW = [
+  {
+    _cls: "fiftyone.core.stages.Limit",
+    kwargs: [["limit", 10]],
+    _uuid: "020b33dd-775a-4b4a-a865-4901e2e6ee43",
```

Comment:
is this is just a hardcoded view to demo changeView()? Can _cls be a list that positionally aligns with kwargs? Or how would you apply multiple stages?

Reply:
Oh yeah that makes sense, I missed the `[]`